### DISCUSSION TITLE
iframes hidden by the current layer still receive click events?

### DISCUSSION THREAD

**Discussion by outcassed** [unpoly/unpoly#454] [General]
I'm using a "cover" layer and it looks like click events are making it through to iframes in the layer underneath me. In this case, there is a Google map hidden by my layer and if it is below my click, it opens up maps.google.com.

I worked around it with
```

up.on(['up:layer:opened', 'up:layer:dismissed', 'up:layer:accepted'], () => {
  document.querySelectorAll('iframe').forEach((iframe) => {
    if (up.layer.contains(iframe)) {
      iframe.style.pointerEvents = 'auto';
    } else {
      iframe.style.pointerEvents = 'none';
    }
  });
});

```

**Comment by triskweline** [comment]
Doesn't the first opaque element receive a click event?

There *is* event bubbling, but that walks across the DOM hierarchy, not visual layers. And that Google Map iframe is not an ancestor of the cover overlay.
